### ISSUE TITLE
Faust source

### ISSUE THREAD

**Issue by cbix** [brummer10/Xmonk.lv2#6]
It would be great to have the xmonk.dsp included in this repo, if possible! Or is it basically the SFFormantModelBP from the Faust libraries? Thanks!

**Comment by brummer10** [comment]
Yes, it is  basically the SFFormantModelBP from the Faust libraries. To make it the monk, I've edited the C++ source by hand.
